SmartDeviceLink/SDLChoiceSetManager.m

```diff
@@ -36,8 +36,10 @@
 #import "SDLRPCResponseNotification.h"
 #import "SDLSetDisplayLayoutResponse.h"
```

Comment:
The `SDLCheckChoiceVROptionalOperation` does not get sent when the `hmiLevel` is non-`NONE` the first time I connect via USB cord. However, if I pull and reconnect the USB cord, then the `SDLCheckChoiceVROptionalOperation` gets sent when the `hmiLevel` is still `NONE`.

Reply:
Fixed; the HMI level wasn't getting reset properly